components/scroll-wrapper/scroll-wrapper.js

```diff
@@ -162,16 +152,16 @@ class ScrollWrapper extends FocusVisiblePolyfillMixin(RtlMixin(LitElement)) {
 	render() {
 		const actions = !this.hideActions ? html`
 			<div class="d2l-scroll-wrapper-actions">
-				<button class="d2l-scroll-wrapper-button d2l-scroll-wrapper-button-left" @click="${this._scrollLeft}" aria-hidden="true">
+				<div class="d2l-scroll-wrapper-button d2l-scroll-wrapper-button-left" @click="${this._scrollLeft}">
 					<d2l-icon icon="tier1:chevron-left"></d2l-icon>
-				</button>
-				<button class="d2l-scroll-wrapper-button d2l-scroll-wrapper-button-right" @click="${this._scrollRight}" aria-hidden="true">
+				</div>
+				<div class="d2l-scroll-wrapper-button d2l-scroll-wrapper-button-right" @click="${this._scrollRight}">
 					<d2l-icon icon="tier1:chevron-right"></d2l-icon>
-				</button>
+				</div>
 			</div>` : null;
 		return html`
 			${actions}
-			<div class="d2l-scroll-wrapper-container" @scroll="${this._checkScrollThresholds}"><slot></slot></div>
+			<div class="d2l-scroll-wrapper-container" @scroll="${this._checkScrollThresholds}" tabindex="0"><slot></slot></div>
```

Comment:
I feel like this is a dumb question, but why does the scroll-wrapper need to be focusable?

Reply:
So that keyboard users have a way to scroll it left/right. The old version of this they simply couldn't scroll it at all. I changed it so that the buttons were focusable but then aXe started complaining that aria-hidden things can't be focusable (the issue Vitalii raised). So just making the scrollable area focusable is the compromise I've arrived at.

Reply:
Ah I see.  So the slot that's within the scroll wrapper could contain other focusable elements, which would mean that we would have focusable elements inside of this focusable element?  I think that might be ok, although I've sometimes run into problems with that sorta thing in the past.  Most recently with nested menus - that scenario is more complicated though because there is a lot of focus management code that tries to redirect focus if focus events fire on focusable parents when clicking on focusable children.  That's just one example of weirdness.  Long story short... I think we should tread carefully.

Reply:
Ok thanks I'll be on the lookout for issues!